### PR TITLE
open-cluster-management/backlog#11774 credential

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: clusters/discovery_credential.adoc [hunk 1]
```diff
@@ -13,14 +13,16 @@ You need access to a https://cloud.redhat.com/[cloud.redhat.com] account.
 [#add-credential]
 == Add your credential
 
-If you have no credentials, see the following process. *Best practice:* Create only one credential in each namespace.
+*Best practice:* Create only one credential in each namespace.
```
Comment (zkayyali812) [inline clusters/discovery_credential.adoc:16]: Perhaps this should state only 1 RHOCM credential in each namespace?

Comment (nathanweatherly) [reply clusters/discovery_credential.adoc:16]: No, there's an edge case where 2 creds in one namespace is acceptable.

Comment (swopebe) [reply clusters/discovery_credential.adoc:16]: ?

**Pull request merged**